\chapter{General discussion, future work and conclusions}
\label{ch:conc}
\acresetall

%Antarctic lakes are exceptional ecosystems that contain microbial life adapted to extreme polar conditions and the local lake geochemistry.
Metagenomics has proven to be an effective way to map the diversity of Antarctic lake ecosystems and provide hints of how they work.
In combination with metaproteomics and abiotic parameters, in-depth descriptions of the ecosystem functions of Ace Lake and Organic Lake were achieved.
The most noteworthy contribution of these studies has been to describe taxa and microbial processes previously unknown to occur in these lakes, and from these descriptions, generate testable hypotheses of population and ecosystem level function.
\section{Possibile future work on Organic Lake}
%The work described in this thesis is part of a program that has pioneering the use of shotgun metagenomics and metaproteomics in Antarctic lakes \cite{Ng2010a, Lauro2011, Yau2011} and the Southern Ocean \cite{Wilkins2012b, Grzymski2012, Williams2012a, Williams2012b}.
To establish a picture of the biotic composition and function of Antarctic lake communities, an observation-driven approach was utilised that allowed for completely new discoveries about these systems to be made.
Bioinformatic pipelines and theoretical models to do this have now been established.
These can be applied to future, similarly observation-based studies of Ace and Organic Lakes to define how they change over time and test our existing models of how the lakes function.
It is also clear that a systems level understanding of the lakes is bolstered by having well-characterised isolates related to members in the community.
Guided by the studies from this thesis, isolation and characterisation of key members of the community, such as \emph{Pyramimonas}, \ac{OLPV}, \ac{OLV}, \emph{Marinobacter} spp. and \emph{Roseovarius} spp. from Organic Lake, could be attempted in the future.
Other complementary techniques, such as single-cell and single-virus genomics and \ac{SIP}, hold promise for learning about specific populations in the community.
Some experiments that could be used to test specific hypotheses generated by the study of Organic Lake are discussed below, along with their expected outcomes and implications.

\subsection{Organic Lake dynamics over time}
To establish a complete understanding of Organic Lake requires descriptions of the microbial community over different time scales.
Currently, a baseline of the microbial community diversity over an annual cycle has not been established for Organic Lake; although certain bacterial taxa have been observed to peak in abundance during summer indicating seasonal fluctuations exist \cite{James1994}.
The next step forward would be to obtain metagenomic and functional data at other points in the year, particularly the winter, to determine how the microbial community changes and copes with the large seasonal changes in temperature, light and ice-cover.
Winter sample in particular can determine how the community persists when the lack of light is expected to curtail photosynthetic production.


Paired metagenomic and metaproteomic analyses of the coastal Southern Ocean from winter and summer has found winter samples were dominated by active chemolithoautotrophs including sulphur-oxidising \emph{Gammaproteobacteria} and ammonia oxidising \emph{Crenarcheaota} \cite{Grzymski2012, Williams2012b}.
If the Organic Lake ecosystem reflects that of the coastal Southern Ocean, the small population of chemolithoautotrophic sulphur-oxidising \emph{Proteobacteria} found to be present (chapter \ref{ch:org}) are expected to become dominant in the winter.
%Succession of the summer bacterial population by \emph{Crenarchaeota} is not expected to occur as no signatures for \emph{Crenarchaeota} or ammonia oxidation were present in Organic Lake.
Metagenomic analysis indicated recycling of reduced nitrogen compounds predominates in Organic Lake and a lack of capacity to form oxidised nitrogen compounds (chapter \ref{ch:org}).
If the model proposed for the Organic Lake nitrogen cycle holds true, we can expect ammonia oxidising microorganisms will not play a part in the community as they do in other similar environments \cite{Voytek1999, Grzymski2012, Williams2012b}.
Another possibility is that some bacterial populations will simply become dormant over the winter, as seems to occur in Artic sea ice communities \cite{Collins2010}.

Photosynthetic nanoflagellates detected in Organic Lake, such as \emph{Pyramimonas} \cite{Bell2003}, are known to be capable of mixotrophy and likely persist over the winter by switching to heterotrophy.
Strategies that might be employed by other photosynthetic algae include utlising starch reserves or developing cyst forms.
Analysis of expression of \acs{RuBisCO} genes over the polar night transition in Lake Bonney showed trends between \acs{RuBisCO} and irradiance levels existed that differed with \acs{RuBisCO} type and depth \cite{Kong2012a}.
As the eucaryotic community was stratified with depth, this suggests that adaptations to absence of light is varies between species \cite{Kong2012a}.
Phototrophic eucaryotes in Organic Lake are likely to have similarly diverse strategies to persist over the winter.
Combined metagenomic and metaproteomic analysis can determine which phototrophic eucaryotes are found during the winter and indicate what metabolic processes they employ in the absence of light.
%In this way, an `-omics' based analysis of Organic Lake community over an annual cycle will shed light on how robust our models of nutrient cycling and concepts of the taxonomic composition are.

%Ideally, metagenomic analyses would be performed on samples taken over a complete annual cycle.
Sampling over the summer season could establish if significant variation in the community exists and the environmental factors driving it.
The large change in the viral composition and abundance between the November and December 2008 samples (chapter \ref{ch:olv}) suggests the shift in the microbial community is related to ice-cover melt and/or increased solar irradiance. 
There were differences in the eucaryotic community when the lake was completely thawed in 2006 (chapter \ref{ch:olv}) and the profile in November 2008 (chapter \ref{ch:org}) when the lake was ice-covered.
Specifically, \emph{Dunaliella}, \emph{Dictyochophyceae}, \emph{Euplotes}, \emph{Dinophyceae} and \emph{Bacillariophyta} were present in both samples, whereas \emph{Pyramimonas}, \emph{Pelagophyceae}, \emph{Pirsonia} and \emph{Caecitellus} was only detected in the December 2006 samples and fungi and \emph{Choanoflagellida} were only in the November 2008 profile (chapters \ref{ch:olv} and \ref{ch:org}).
It is unclear whether the the variation between the summer samples, December 2006, November 2008 and December 2008 are due to differences in ice-cover, sampling locations  (2006 and December 2008 samples were littoral, while November 2008 was over the deepest point of the lake) or other environmental variables.
Metagenomic sequencing of a summer time course would provide a high resolution view of changes in the microbial community over the season.
This can act as a bench mark to compare back to the summer 2006 and 2008 samples indicating if the differences that have been observed fall in normal range of variability over a season.

Some members of the population appear to be persistent over long time scales.
For example, \emph{Dunaliella} \cite{Franzmann1987b}, choanoflagellates \cite{vandenHoff1986}, \emph{Marinobacter}, \emph{Psychroflexus} \emph{Roseovarius} and \emph{Halomonas} \cite{Bowman2000b} have been recorded in Organic Lake previously and were detected again (chapters \ref{ch:olv} and \ref{ch:org}).
How they vary throughout the year and between years is unclear.
Organic Lake has been shown to be physically variable over a decadal time scale with changes in water level of $\sim$1 m leading to large changes in the water column structure \cite{Gibson1995, Gibson1996}.
Notably, the pycnocline in Organic Lake occurred at 5.7 m in 2008; much lower than 3.5 m where it first reported to occur in 1978 \cite{Franzmann1987b}.
If the lake completely mixes, this would challenge the oxygen-senstitive microbes and processes occurring in the deep zone.
The large physical changes suggests that the microbial community would similarly display a high degree of variation between seasons.
Substantial differences were observed in \emph{cbbM} gene copy number and vertical distribution between three field seasons in Lake Bonney \cite{Kong2012b} that suggests Antarctic lake microbial communities may be quite variable over time.
An idea of the microbial diversity over an annual cycle would provide an indication of the variability of the community.
When compared back to the summer samples from 2006 and 2008, an idea of how robust the populations are over the years.
%Long-term sampling of Southern Ocean coastal surface waters suggests the community has a reproducible annual cycle \cite{Murray2007}.

In chapter \ref{ch:olv}, two near-complete \ac{OLPV} genomes and a complete \ac{OLV} genome were described.
Genomic analysis indicated \ac{OLV} is a new member of the virophage virus family, which obligately utilise a  giant `helper' virus to complete their replication cycle but impair their helper's infectivity in the process.
The likely helper virus of \ac{OLV} is \ac{OLPV} and the host was inferred to be the unicellular alga, \emph{Pyramimonas}.
The inferred interaction of the \ac{OLV}, \ac{OLPV} and host was used to generate a Lotka-Volterra model of their population dynamics that showed if \ac{OLV} acts as a `predator of a predator', this would lead to increased frequency of population cycling.
Sampling intensively over a short time scale in the summer will allow us to test the Lotka-Volterra model of the \ac{OLV}, \ac{OLPV} and host algae population.
In the model, the density of each population oscillates in a phase-shifted manner in the sequence: prey, predator and predator of predator.
By observing the natural dynamics of these populations, we can determine if they fit with the proposed model.
If they do fit, the observed population densities can be used to derive the parameters that govern the interaction.
Close observation of these three taxa can suggest if additional modifications to the existing model or alternative models would better fit the data.

\subsection{\acs{OLV} physiology and ecology}
Since the discovery of the \ac{OLV}, two other members of the virophage family have been reported that have afforded a new perspective on \ac{OLV}.
The first of these is the Mavirus (for Maverick virus) so named for its evolutionary relationship with the Maverick/Politon class of eucaryotic transposons \cite{Fischer2011a}.
Like Sputnik, Mavirus has an absolute requirement for a helper virus, \ac{CroV}, to replicate and is deleterious to its helper \cite{Fischer2011a}.
The other virophage reported was called Sputnik 2 (its genome is almost identical to Sputnik), but unlike Sputnik, it was found both as a separate genome and integrated in its helper lentille virus \cite{Desnues2012}.
These two virophage systems in culture are associated with heterophic protists making \ac{OLV} the only genomic sequence currently available for virophage affecting cosmopolitan phytoplankton species.
Therefore isolating \ac{OLV}, acquiring genomes of \ac{OLV} relatives in Antarctic lakes and determining fundamental properties of \ac{OLV} physiology and dynamics would contribute immensely to our understanding of the evolution and diversity of virophages in general and is highly relevant to other aquatic systems.

The evidence that Mavirus has the same `virophage' phenotype as Sputnik strengthens the inference that \ac{OLV} does as well.
This is in part because Mavirus and \ac{CroV} and quite divergent from Sputnik and mimivirus respectively, yet retain the same traits \cite{Fischer2010, Fischer2011a}, suggesting a common feature of the whole lineage.
Moreover, as both mimivirus and \ac{CroV} encode much of their transcriptional machinery, they do not localise to the nucleus during infection but generate the viral factory in the cytoplasm of their host \cite{LaScola2008, Fischer2011a}.
Sputnik and Mavirus replicate in the giant virus factory making use of helper virus' replication and transcription systems, not the host cell's \cite{LaScola2008, Claverie2009, Fischer2011a}.
In this way, the replicative strategy of the mimivirus lineage makes them vulnerable to virophages parasitising necessary resources during replication, which is a likely cause of reduced helper virus production \cite{Claverie2009, Fischer2011a, Fischer2011b}.
As \ac{OLPV} encodes all \textsc{RNA} polymerase subunits (see GenBank accession HQ704802), this is evidence that it replicates solely in the cytoplasm, thereby making it susceptible to a virophage.
A larger census of virophage and helper viruses would be able to show if virophages only associate with members of the \ac{NCLDV} clade that replicate in the cytoplasm.
Ultimately, definitive confirmation of a detrimental effect on the helper by \ac{OLV} co-infection is likely only possible by observing infection in culture.

Although Sputnik and Mavirus have similar effects on their helper viruses, their infection strategies appear different raising interesting considerations for \ac{OLV}.
%All three genomes have share homologues of the Sputnik V20 \ac{MCP}, V3 FstK-HerA DNA packaging ATPase and V9 putative cysteine protease but otherwise are quite divergent.
%Sputnik and Mavirus additionally share a the V13 primase/superfamily 3 helicase and the V14 Zn-ribbon domain containing protein.
%While Sputnik and \ac{OLV} share the V18/19 minor virion protein, V1 primase-polymerase containing protein and V21 hypothetical protein.
Mavirus can be independently phagocytosed by \emph{Cafeteria roenbergensis} in the absence of \ac{CroV} -- perhaps serving as a defence for the host in the case of \ac{CroV} infection \cite{Fischer2011a}.
In support of this, Mavirus possesses a retroviral-family integrase that is theorised to have been separately acquired as a way to stabilise the relationship between the ancestral Mavirus and its cellular host \cite{Fischer2011a} although as yet, Mavirus has not been reported to be integrated in \emph{Cafeteria roenbergensis}.
In contrast, Sputnik seems to associate directly with the fibrils that coat the mimivirus virion, not the host \emph{Acanthamoeba} cell \cite{Boyer2011}.
These two modes of infection, that is, a virophage-bearing host being infected by a giant virus \emph{vs}. a host being infected by a virophage-bearing giant virus, produces different selection pressures that would lead to distinct effects on the population dynamics in the environment.
Detection of complete \ac{OLV} genomes in the 0.8--0.1 \textmu{}m size fraction indicates it was captured in association with larger particles but does not distinguish how it is transmitted.

Determination of the mode of infection for \ac{OLV} would improve predictive modelling of \ac{OLV} driven impacts on algal blooms.
This could be achieved by observation of \ac{OLV}-\ac{OLPV} infections in culture.
However, recently developed methods for flow cytometric sorting to obtain \acp{SAG} or \acp{SVG} \cite{Martinez-Martinez2011, Allen2011} could be applied to Organic Lake water samples to study the mode of \ac{OLV}, \ac{OLPV} and host interaction.
For example, a sample population of single host algal cells could be fluorescence sorted and screened for the presence of \ac{OLPV} and \ac{OLV}.
At the same time, a sample population of single \ac{OLPV} particles could be sorted and screened for the presence of \ac{OLV}.
This could reveal if \ac{OLV} is able to associate with the host independently of \ac{OLPV} or \emph{vice versa}.
Examination of the infected algal cells could also establish fundmental properties of the algae and virus populations such as proportions of infected cells and proportions of \ac{OLPV} infections that include an \ac{OLV}.
As these methods are quite new, an experiment of this kind would require optimisation to successfully capture an adequate sample of infected cells and target giant viruses.
However, just obtaining \acp{SVG} would provide invaluable information on virus diversity and genetic content making it worthwhile pursuing as a complement to metagenomic sequencing.

A final consideration for \ac{OLV} and \ac{OLPV} dynamics is suggested by the discovery that Sputnik 2 can integrate into its helper lentille virus \cite{Desnues2012}.
Integration of Sputnik 2 is localised to a 352 bp region corresponding to the Sputnik V6 gene that encodes a collagen-like repeat-containing protein shared by Sputnik 2, lentille virus and mamavirus  \cite{Desnues2012}. 
\ac{OLV} and \ac{OLPV} share a homologous collagen-like repeat-containing protein that may similarly function as a site of integration.
As yet, the conditions and mechanism by which Sputnik 2 integrates is unknown.
Screening of metagenomic assemblies, \acp{SVG} or isolated \ac{OLPV} genomes for integrated virophages can determine if this occurs in the environment.
Integration of \ac{OLV} and \ac{OLPV} could have interesting evolutionary functions.
One possibility is that integration of virophages may function analogously to lysogeny in bacteriophages if integration leads to virophage inactivity.
In this scenario, virophages would integrate into their helper virus when helper virus densities are low to avoid driving their helpers to extinction.
On the other hand, if integration does not entail dormancy of the virophage, it could function as a means to ensure transmission.
In either case, evidence of integration can be found in future genomic studies of Organic Lake.

\subsection{Organic lake biogeochemistry}
The models of carbon, nitrogen and sulphur cycling in Organic Lake were constructed based on the presence and abundance of known marker genes  along with data of the lake's chemical properties.
As yet, metaproteomic analyses from the same samples have not been performed, but this would help corroborate the inferred pathways are active. 
Other inferrences of Organic Lake biogeochemical function can be specifically tested on organisms in culture, tested by measuring rates of reaction \emph{in situ} or supported with additional molecular and chemical analyses.

For the carbon cycle, it was hypothesized that \ac{AAnP} and rhodopsin mediated photoheterotrophy can conserve carbon for use in biosynthesis  reducing overall carbon loss.
The conditions under which \ac{AAnP} is active can be tested on related organisms in culture, for example \emph{Roseovarius tolerans} from Ekho Lake in the Vestfold Hills. 
It is already known for \emph{R. tolerans} constant dim light suppresses \ac{BchlA} production while darkness stimulates it \cite{Labrenz1999}.
\emph{R.tolerans}, or related isolates, can be grown under a larger range of conditions to determine when \ac{AAnP} becomes an active process.
This might include further varying light intensity and cycle duration; varying organic carbon concentration and oxygen concentrations.
To determine the contribution of light to growth, the difference in growth when \ac{AAnP} is active and when it is not can be compared.
Characterising \ac{AAnP} in a model isolate from a similar Antarctic lake can inform an understanding of how it functions in Organic Lake.
However, to gain an understanding of when \ac{AAnP} is active \emph{in situ}, levels of \ac{BchlA} at different points in the year can be measured to see it correlates with seasonal changes.
In the case of the influence of rhodopsin-mediated phototrophy on the Organic Lake system, the function of rhodopsins first needs to be characterised in the diverse organisms in which they are found to confirm they are indeed used in photoheterotrophic growth.
In marine \emph{Flavobacteria}, there is already evidence that light stimulates growth through the action of rhodopsins \cite{Gomez-Consarnau2007}.
The same experimental design used to establish this can be applied to \emph{Psychroflexus gondwanensis} isolated from Organic Lake, \emph{Marinobacter} sp. ELB17 and \emph{Octadecabacter antarcticus}, which are relatives of species found in Organic Lake.

In the model of the nitrogen cycle, nitrogen fixation was inferred to be negligible due to the high concentrations of ammonia while denitrification was assumed to be limited by low levels of oxidised nitrogen compounds and lack of potential for nitrification to regenerate them.
In these cases where rates of reaction were inferred to be slow although genetic potential for them exists, the reaction rates can be directly measured.
Nitrogen fixation and denitrification rates can be measured from Organic Lake water samples by incubation with $^{15}$N labelled substrates or acetylene block assays.
Similarly, it was inferred that carbon fixation rates from photosynthetic algae and chemolithoautotrophic bacteria were lower than respiration and fermentation based on genetic potential.
As mentioned previously, populations of chemolithoautotrophs may increase in the winter, potentially becoming signficant source of primary production.
Another consideration for the carbon budget is how sustained levels of chemolithoautotrophy throughout the year compares to the short burst of photosynthetic production over the summer.
This appears to be the case in Lake Bonney in McMurdo Dry Valleys as \acs{RuBisCO} linked to chemolithoautrophs remained constant over the polar night transition \cite{Kong2012b}.
Measuring rates of primary production by $^{14}$C incorporation and respiration rates at different points in the season can acertain if there truly is a shortfall in the carbon budget in Organic Lake, estimate how large it is and pinpoint if the main source of primary production is from photo- or chemoautotrophy.

Both the carbon and nitrogen cycles were also constructed with the assumption that inputs are also negligible.
For instance, denitrification rates would be limited by low fixation coupled with low rates of nitrification.
Nonetheless, it is possible that sufficient nitrate inputs occur that would sustain denitrification thereby by-passing the need for endogenous nitrification.
Monitoring for presence of melt streams that might feed into Organic Lake and determining their chemical profiles can gauge the contribution of allochtonous carbon and nitrogen to Organic Lake.

The unusually high concentrations of \ac{DMS} in the bottom waters of Organic Lake were inferred to originate from bacterial lysis of \ac{DMSP}.
As \ac{DMSP} lyases have only been discovered fairly recently \cite{Todd2007, Curson2008}, they have only been characterised from a few isolates \cite{Todd2007, Curson2008, Curson2010, Todd2010, Curson2012}.
Confirmation that the homologues found in Organic Lake are indeed functional can be achieved by cloning into the expression vector system established by \citet{Todd2007} and assaying for \ac{DMSP} lyase activity.
This would also provide valuable insight into the \ac{DMSP} lyases that are most relevant to cold and hypersaline environments.

Although it was inferred that \ac{DMSP} lysis was the main source of \ac{DMS} in the bottom zone of Organic Lake, production of \ac{DMS} other by anaerobic pathways are also possible.
Furthermore, \ac{DMS} was inferred to accumulate due to slow rates breakdown in the bottom zone.
Incubating Organic Lake water samples with radio-labelled \ac{DMSP} and tracking production of volatile \ac{DMS} would confirm the lysis pathway is an active process and determine the rates of reaction.
Performing this assay on water from different depths could determine if concentration of \ac{DMS} is high in the bottom zone because it is being produced there at a higher rate.
A similar assay can be performed using labelled \ac{DMS} to show if \ac{DMS} can be broken down and if this differs with depth.
If the proposed model for sulphur cycling based on marker gene frequencies is correct, \ac{DMS} breakdown in the bottom samples should be slow or not occur.
While this experiment will not exclude the possibility that \ac{DMS} is produced by an alternative pathway, knowing the rates of \ac{DMSP} lysis and \ac{DMS} breakdown and the \ac{DMS} concentration in a sample can indicate the existence other sources of \ac{DMS} production.

Most of the Organic lake \ac{DMSP} lyase genes could be linked to a taxonomic group except the most abundant type, OL-dddD, which had indication of both \emph{Alpha}- or \emph{Gammaproteobacteria} origins.
Identifying the organisms involved in \ac{DMSP} lysis can be achieved with \ac{SIP}.
This would involve incubation of $^{13}$C-labelled \ac{DMSP} in Organic Lake water samples followed by sequencing of the labelled DNA.
Sequencing of the \ac{SSU} genes would determine which specific taxonomic groups were the main contributors to \ac{DMSP} lysis and screening for \ac{DMSP} lyase genes would identify which \ac{DMSP} lyase is involved.
Performing this experiment at different depths of the water column could determine if the same organisms are responsible for \ac{DMSP} lysis at different depths.
No marker genes yet exist for \ac{DMS} oxidation although it can be readily utilised as a growth substrate by diverse microorganisms \cite{Johnston2008}.
The same experiment can be performed using $^{13}$C-labelled \ac{DMS} to determine the taxa involved in breaking it down in different depths of the lake.
In conjunction with metagenomic sequencing, or flow sorting and \acp{SAG} of \ac{DMS}/\ac{DMSP}-degraders, the biochemical pathways involved in \ac{DMS} breakdown can then be reconstructed giving a more complete understanding of Organic Lake sulphur biogeochemistry.

\section{Perspectives on `-omics' approaches }
It was not so long ago that what could be called the second molecular revolution in microbial ecology began in earnest with the first shotgun metagenomic sequencing of a marine virome \cite{Breitbart2002}.
That metagenome, sequenced with Sanger sequencing technology, was a modest 1.28 Mbp \cite{Breitbart2002}.
The first shotgun sequenced metagenomes of cellular life from the Sargasso Sea \cite{Venter2004} and the Iron Mountain acid mine drainage \cite{Tyson2004} added 265 Mbp and 76 Mbp respectively to the public databanks.

\subsection{Next generation sequencing technologies in metagenomics}
Since the availability of high-throughput \ac{NGS} technologies, the volume of metagenomic data has increased exponentially and is continuing to rise.
Currently, the Genomes Online Database (\url{http://www.genomesonline.org}) lists 2,350 metagenomic samples from 369 separate projects.
To put it in another perspective, a single sample from the Antarctic lake datasets described in this thesis sequenced by the Roche GS-FLX titanium sequencer is 140 Mbp, while a recent study using Applied Biosystems SOLiD sequencing of a marine sample \cite{Iverson2012} analysed 55,000 Mbp -- close to 400 times the amount of data.
This illustrates how sequencing technology is advancing so rapidly that each new study has to develop new ways of looking into the microbial milieu captured in the particular sample.
Current \ac{NGS} sequencing platforms are shown in \tabreft{tab:seq_tech} comparing sequence length, throughput and error rates.
Clearly, there are trade-offs between each type with higher read depth technologies offering shorter read lengths. 
Application of sequencing technologies that offer greater read depths in future Antarctic lake studies has the potential to reveal rare members of the community.
For example, a recent study compared the community profile over time using a conventional sequencing effort and deep sequencing.
Deep sequencing revealed members of the community that appeared to seasonally become absent in the conventional sequencing set actually were constantly present at very low abundance. %%%Find this citation
%The long read length offered by single-molecular sequencing (Pacific Biosciences) is off-set by a high error-rate and has not as yet been applied to metagenomics.
\input{conc_figures/seq_tech.tex}

\subsection{Emerging bioinformatic bottlenecks}
With the higher throughput and the falling costs of sequencing, genomic projects are now experiencing `bioinfomatic bottlenecks' where computational analysis has become a significant challenge (reviewed by \citet{Scholz2012}).
This is because availability of computational resources and scaling-up of algorithms to accomodate more data or computing clusters is not occurring as quickly as the accumulation of sequence data.
Metagenomic sequencing projects typically entail computationally intensive \acs{BLAST}-like comparisons before biological interpretations can be made.
Local cluster computing used in this thesis has been a standard way to process metagenomic datasets.
However, the annual cost of acquiring, running and administering a standard rack mount server has been estimated to be \$2,160--\$5,160 US per node \cite{Wilkening2009}.
This can in some cases work out to be more expensive than the cost of sequencing itself.
Certainly the time to acquire sequence data once samples have been collected is much shorter than the time needed to analyse it.
Without increases in compute resources, the time taken to process massive \ac{NGS} datasets will become prohibitively slow and is a consideration for future metagenomic studies on the Antarctic lakes.
For example, analysis of metagenomic data produced by Illumina technology using a standard pipeline was calculated to take decades on a single processor or weeks to months on 1,000 \acs{CPU}s \cite{Evanko2009}.
Ultimately, for computational analysis to catch up with advances in sequencing technology requires more efficient processing, faster processors and/or faster algorithms.

\subsection{Webserver analyses}
A shortage in computational resources can be addressed to some extent by the use of webserver pipelines specialised in metagenomic processing.
These include \ac{MG-RAST} \cite{Meyer2008}, \ac{CAMERA} \cite{Sun2011}, \ac{IMG/M} \cite{Markowitz2008, Markowitz2012}, EBI Metagenomics \cite{Hunter2012}, as well as \textsc{Metavir} \cite{Roux2011} and \ac{VIROME} \cite{Wommack2012} for viral metagenomes.
These webservers take on the burden of maintaining necessary programs, storage of sequence databases and provide computational power.
Each have unique features that may be more desirable for particular sample types, analyses or user preferences.
One example of this is the different functional analyses offerred by EBI Metagenomics and \ac{MG-RAST}, which implement InterPro and SEED subsystems respectively.

Webservers were not extensively utilised in this this thesis apart from \acs{BLAST} searches for specific genes of interest against the \acs{GOS} metagenomes.
Use of \ac{CAMERA} was convenient for broad searches of \ac{OLV} against all other metagenomes (chapter \ref{ch:olv}) as it saved from having to download large metagenomic datasets for a local \acs{BLAST}.
However, searching for genes involved in photoheterotrophy and \ac{DMSP} catabolism in Antarctic and \acs{GOS} metagenomes (chapter \ref{org} required multiple \acs{BLAST} comparisons against a smaller database.
In this case it was much easier to download the necessary metagenomes and run the searches on the local cluster.
Webserver workflows have the disadvantage that they are not as readily customisble.
For example, current webservers have been designed towards analysis of bacteria/archaea or viral sequences; although the metagenomes from this thesis contained significant proportions of both, as well as eucaryotic sequences.
To take advantage of the range of specialist features offered by different pipelines can introduce inefficiencies such as performing similar sequence quality control steps in order to be able to fit within their distinct analytical schemas. 
The different outputs then need to be combined locally and may not be directly comparable to one another due to differences in processing.
However, it would be worthwhile to trial use of webservers in the future to utilise their specialist tools and to relieve some pressure on local computing time.
%Processing massive datasets may still take an unfeasibly long time depending on the capabilites and usage policies of the webserver.
%Therefore, they are not a final solution for improving processing time but rather convenient tools and repositories.

\subsection{Cloud computing and \acs{GPU}s}
Cloud computing, the use of computing infrastructure, software or platforms as a service, looks to be a promising way to make up the shortfall in processing power by affording increases in computational efficiency (reviewed by \citet{Schatz2010} and \citet{Thakur2012}).
Several major providers exist: the largest publically available service is Amazon's \ac{EC2} and the Department of Energy's Magellen Cloud is available for academic use.
Unlike webservers, cloud computing can be used for any task, but require more expertise.
Gains in efficiency for the user can occur because clouds provide access to computing power that varies `elastically' with the demands of the task without the cost of maintaining a local cluster.
These infrastructure costs are more effectively absorbed by large providers than by smaller facilities allowing clouds to potentially provide the same resources at lower cost.
Also, by having access to more computational nodes, programs like \acs{BLAST} can be run in parallel increasing the throughput of analysis.

Whether it works out to be more cost efficient in practice depends of the expected volume of data and frequency of analysis \cite{Wilkening2009}.
The cost--benefit appears greatest for those who expect sporadic use or need to augment computer resources on an unpredictable basis \cite{Wilkening2009}.
To benchmark the costs and time involved in analysis of metagenomic data on the \ac{EC2}, \citet{Angiuoli2011} performed a cluster analysis and \acs{BLAST} comparison of 631 Mbp of metagenomic sequence (454 GS FLX titanium) against the RefSeq and \acs{COG} databases.
This analysis finished in a little over 6 hours using a maximim of 160 \acs{CPU}s and at a total cost of \$56 US \cite{Angiuoli2011}, giving an indication of how clouds might be leveraged for metagenomic processing.
However, not all computational tasks see improved processing times when run on a cloud.
Additional considerations include adequate connection speeds to upload large quantities of data, data security, and data management.
With these considerations in minde, use of cloud resources looks be most useful in future Antarctic projects to run \acs{BLAST} searches and potentially \emph{de novo} (discussed below).
%This is starting to change as developers are packaging metagenomics specific applications for specific use in clouds.
%Caparaso paper, trying to make cloud computing easier for newbies.
%Currently, improvements in technology is increasing sequencing throughput at a rate of five fold per year outstripping improvements in computer processor speed, which double every 18 months \cite{Schatz2010}.
Cutting down processing times of \ac{NGS} analysis could be achieved by ultilising hardware-driven solutions.
The most promising of these is use of \acp{GPU} instead of conventional \acp{CPU}.
A trial of use of \acp{GPU} has reported 10--15 fold increase in processing speeds.
Currently, their use seems to be limited by the need to develop software for bioinformatics that can run on the different architechture.
% from Hunter, \cite{Sun} %note that they have a GPU cluster on barrine.
This has already been achieved for \acs{BLAST} analysis which will undoubtedly speed up processing time of metagenomic studies.
%Find this example

\subsubsection{Bioinformatic tools for \acs{NGS} metagenomic data}
While it is possible that a break-through computer algorithm emerges that can process greater amounts of data using the same amount of processing power, this sort of advance will be difficult to foresee.
\ac{NGS} sequencing technology is spurring on development of computational tools and approaches to be run on multiple processors in parallel thereby decreasing processing time.
Metagenomic analyses are typically based on data from single reads, such as was described in chapter \ref{ch:org} or from assembled genomic information, for example the work described in chapter \ref{ch:olv}.
These analyses entail two tasks in particular that are unique to metagenomics: classification of short sequences and assembly from mixed species datasets. 
An exhaustive listing of all bioinformatic tools that accomplish these task is not possible so discussion will be restricted to the most recent developments.
%Many tools initially developed to handle Sanger or 454 data now must be scaled up to accomodate the scale of Illumina sequencing \tabref{tab:seq_tech}.
\subsubsection{Classification of short reads}
Some of the challenges faced for classification of short reads with \ac{NGS} data is differences in biases, but the most challenging factor is the large volume of data that necessitates finding ways to avoid, or reduce the amount of BLAST-like comparisons.
Several approaches have been attempted to classify sequences that can roughly be divided into clustering-based methods, homology methods and intrinsic sequence properties.
New algorithms are being developed to meet these new demands such as \textsc{Genometa} \cite{Davenport2012}.
%Tools for binning include ClaMS \cite{Pati2011}, 

\subsubsection{Metagenomic assembly}
The greater amount of sequence per run provided by Illumina, Ion Torrent and SOLiD technologies allows more genomic information to be extracted and potentially more whole genomes from environments.
These technologies were not designed with metagenomics in mind so their sequencing algorithms don't necessarily take into account the complex mixture.
One example of this is a newly released sequence of a Euryarchaeon genome assembled from SOLiD sequence implementing a soon to be released program SEAStar.
%Find examples from the Ion, Titus Brown Iowa corn with k-mers and Illumina. De brujin graphs.
% The benefit of the greater sequencing depths means more genomes can be potentially closed.
% Could a metagenome ever be really `closed' metagenome, unless it is the most simple community, it seems difficult to imagine.

Now that there are many metagenomes available from a whole range over environments, comparing metagenomes to one another can yeild insights into patterns between environments.
It can also give some sort of environmental designation to otherwise unknown sequences that have no matches from genomic databases of isolate.

%Cloud computing also might facilitate scaling up of bioinformatic tools. cite caparaso 
%Efficiency of the analytical tasks can be achieved by less redundancy analysis and effective data sharing.

\subsection{ 'omes are good as your database and knowledge base}
Aside from the computational limitations, `-omics' type analyses rely a great deal on current sequence databases and laboratory studies to make meaningful inferences.
For example, detection of the \ac{OLV} was contingent on the presence of a sufficiently close relative, Sputnik \cite{LaScola2008}, in the non-redundant database.
%However, a large proportion of metagenomic sequence from viruses have no matches in any sequence databases and thus are not assignable to taxonomic or functional groups \cite{Lopez-Bueno2009}.
Similarly, the recent sequencing and characterisation of \ac{DMSP} lyase genes enabled them to be detected in the Organic Lake data.
%Conversely, it also means that a great deal of the data is remains unknown.
%Make point about being a repository for the future.

A dependency on sequence databases applies even more to metaproteomic analysis.
Unlike metagenomics where fairly distantly related sequences can be matched to a metagenome to give some idea of its biological significance, spectral matching fails if protein sequences with high identity to those in the sample are not available.
This makes metaproteomics much more reliant on sequence database availibility and accuracy.
Furthermore, identification of proteins only indicates that they are expressed, but characterisation of the same or closely related protein is still necessary to infer its function in the sample.
This exemplifies how both metagenomics and metaproteomics need to be paired with basic laboratory studies for meaningful biological inferences to be made.

\subsection{Metagenomes and metaproteomes are time capsules}
Conversely, it means a great deal of metagenomics sequences and metaproteomic mass spectra data remains to be taxonomically or functionally assigned.
The \textsc{DNA} sequence and mass spectra data that has been produced form a lasting record of the state of the microbial communities at the point in time that they were collected.
It thus acts as a resource against which other datasets can be compared to gain an understanding of variability between ecosystems and as a bench mark to gauge changes overtime.
The data can also be specifically mined to recover features of interest, for example sequences that encode enzymes with desired activities or bioactive compounds can be recovered by synthesis from the sequence information.
As sequence databases grow and more of the microbial world is characterised, this archive of molecular data can be re-analysed to learn even more from these unique ecosystems.



\subsubsection{Where to from here?}
As mentioned above,
%cite your future work section and other people's commentaries.
 future `-omic' studies need to now encompass longer time scales, finer spatial resolutions and focussed analysis of populations and individuals that make up communities.
This will allow for relationships to become apparent between molecular data and environmental parameters over space and time.
Pairing `-omics' tools with complementary techniques designed to target specific individuals or populations, community functions and biogeochemical processes is a promising way to enable an even deeper understanding of ecosystems by combining their respective strengths.

The hypotheses built on environmental data can then guide laboratory research studies and thus feedback into supporting a systems-level understanding of microbial communities.

%Cite \cite{Warneke2007}
%\figref{fig:prok_diversity} illustrates how much additional bacterial and archaeal taxonomic diversity has been revealed by environmental sequencing and also suggests how much more of their genomic diversity is still undescribed. 
%\input{conc_figures/prok_diversity.tex}

\section{Concluding remarks}

This thesis has demonstrated how metagenomics and metaproteomics can be used as a tool for characterising Antarctic lake ecosystems.
The \textsc{DNA} sequence and mass spectra data that has been produced form a lasting record of the state of the microbial communities at the point in time that they were collected.
It thus acts as a resource against which other datasets can be compared to gain an understanding of variability between ecosystems and as a bench mark to gauge changes overtime.
%The data can also be specifically mined to recover features of interest, for example sequences that encode enzymes with desired activities or bioactive compounds can be recovered by synthesis from the sequence information.
As sequence databases grow and more of the microbial world is characterised, this archive of molecular data can be re-analysed to learn even more from these unique ecosystems.


%\section{Summary of outcomes from this work}
%This has necessitated the development of new ways to extract biological inferences from complex datasets.
%The overall objective of this thesis was to study Antarctic lake ecosystems using an integrative metagenomics-driven approach.
%In order to do this, methods were developed (chapter \ref{ch:ace}) to measure properties of the community that would work alongside metagenomic sequencing to piece together a comprehensive description of ecosystem structure and function.
%An epifluorescene microscopy protocol was designed that makes use of 0.01 \textmu{}m pore-size \ac{PCTE} membrane filters.
%Using this protocol, cells and \acp{VLP} were successfully visualised and enumerated along the depth profile of Ace Lake, and subsequently in Organic Lake.
%The need to develop this method was due to the discontinuation of supply of Anodisc filters, which are the standard filter used for direct counts of \acp{VLP} by epifluorescence microscopy.
%Although cell and \ac{VLP} densities are fairly basic properties of the community, their measurement nonetheless revealed surprising results, such as the lack of correspondence between turbidity and cell density in Organic Lake.

%A bioinformatic pipeline for analysis of metaproteomic mass spectra was also developed (chapter \ref{ch:ace}) that incorporated use of matching metagenomic databases for protein identification and spectral counting to estimate protein abundances.
%When applied to the metaproteomic analysis of Ace Lake, this workflow showed statistically significant functional differences between the strata of the lake.
%Metaproteomic analysis also revealed the functional capacity of dominant populations in the mixolimnion.
%The incorporation of genomic and functional data in the study of Ace Lake provided a framework for approaching subsequent studies on Organic Lake.

%In chapter \ref{ch:olv}, two near-complete \ac{OLPV} genomes and a complete \ac{OLV} genome were described.
%Genomic analysis of \ac{OLV} indicated it was a new member of the virophage virus family, which obligately utilise a  giant `helper' virus to complete their replication cycle but impair their helper's infectivity in the process.
%Comparison of \ac{OLV} and \ac{OLPV} genomes identified shared regions that suggest \ac{OLV} is a virophage of \ac{OLPV}.
%The metaproteomic analysis identified capsids from both viruses confirming members of the population were actively replicating viruses and not, for example, accessory genetic elements.
%The availibility of \textsc{DNA} sequences from the whole community meant the most likely algal host, \emph{Pyramimonas}, could be identified.
%The inferred interaction of the \ac{OLV}, \ac{OLPV} and host was used to generate a model of their population dynamics that showed if \ac{OLV} acts as a `predator of a predator', this would lead to increased frequency of population cycling.
%An increased frequency of algal blooms would lead to an overall increase in organic carbon released over the summer season and thus incease secondary production.
%Potential relatives of the \ac{OLV} were also found in Ace Lake and other aquatic environments indicating virophages may play a role in regulating host--virus population dynamics in other ecosystems.

%In chapter \ref{ch:org} a profile of the Organic Lake microbial taxonomic composition and potential for nutrient cycling was determined.
%The community was found to consist primarily of eucaryotic algae related to \emph{Dunaliella} and three heterotrophic bacterial genera: \emph{Marinobacter}, \emph{Roseovarius} and \emph{Psychroflexus}.
%The suboxic bottom zone of Organic Lake additionally contained lower abundance populations of \emph{Firmicutes}, \emph{Deltaproteobacteria} and \emph{Epsilonproteobacteria}.
%Examination of genes involved in carbon cycling indicated a net loss of carbon could occur as potential for fixation was lower than for respiration.
%However, genes for photo- and lithoheterotrophic pathways linked to the dominant bacterial lineages were abundant; in particular \ac{AAnP} genes were much higher than in any other aquatic environment surveyed.
%Use of energy sources apart from organic carbon was hypothesized to be a specific adaptation of the heterotrophic bacterial population to conserve carbon within the system.
%The capacity for nitrogen cycling also showed a shift away from fixation and a predominance of recycling of reduced nitrogen compounds.
%This was similarly hypothesized to function as a mechanism to limit nitrogen loss.
%A targeted search for genes involved in \ac{DMS} conversions found an abundance of \ac{DMSP} lyase genes indicating lysis of \ac{DMSP} is the likely source of the high \ac{DMS} levels that has been detected in Organic Lake.
%\ac{DMSP} lyase genes were linked to \emph{Gammaproteobacteria} and \emph{Alphaproteobacteria} indicating \ac{DMSP} is an important carbon and energy source for these bacteria.
%Unlike marine environments, \ac{DMSP} lyase genes were more abundant than \ac{DMSP} demethylase genes. 
%The only other metagenomes to show comparably high abundance of \ac{DMSP} lyase genes were other hypersaline environments suggesting the DMSP lysis pathway is favoured in high salinity systems. 
%We have used theory to guide our work.

