\chapter{General discussion, conclusions and future work}
\label{ch:conc}
\acresetall

\section{Perspectives on `-omics' approaches }
It was not so long ago that what could be called the second molecular revolution in microbial ecology began in earnest with they first shotgun metagenomic sequences of a marine virome \cite{Breitbart2002}.
That metagenome, sequenced with Sanger sequencing technology, was a modest $\sim$ 1.28 Mbp \cite{Breitbart2002}.
The first shotgun sequenced metagenomes of cellular life from the Sargasso Sea \cite{Venter2004} and the Iron Mountain acid mine drainage \cite{Tyson2004} added 265 Mbp and 76 Mbp respectively to the public databanks.

Since the availability of next generation high-throughput sequencing technologies, the volume of data has increased exponentially and is continuing to rise.
A single sample from the Antarctic lake datasets described in this thesis sequenced by the Roche GS-FLX titanium sequencer is 140 Mbp, while a recent study using Applied Biosystems SOLiD sequencing of a marine sample \cite{Iverson2012} analysed 55,000 Mbp -- close to 400 times the amount of data.
This illustrates just how quickly sequencing technology advancing that each new study essentially has to develop new ways of looking into the microbial milieu captured in their particular dataset.
The benefits of using high-throughput sequencing technologies means that shotgun metagenomics will continue to be used as tool in the foreseeable future.
But it also means encountering tough analytical challenges that comes with it.

With higher throughput and falling costs of sequencing, genomic projects are finding analytical bottle necks are occuring in computational analysis.
This is because availability of computational resources and scaling-up of algorithms to multiple genomes or computing clusters is not occurring as quickly as the amount of sequencing.
Metagenomic sequencing projects entail a minimum level of processing that includes data clean-up, assembly, \ac{ORF} prediction, clustering and generally \ac{BLAST} like comparisons before biological interpretations can be made.
The most intensive of these is the sequence comparisons to known databanks.
With the sheer volume of next generation sequencing datasets, this type of analysis requires a supercomputer to process.
Local cluster computing has thus far been a standard way to process metagenomic datasets. 
Acquiring the necessary computational resources can work out more costly for groups expecting only the sporadic use.
The volumes of data can also overwhelm the capabilities of single computation clusters \cite{Iverson2012}.
This has been addressed to some extent by use of webserver pipelines specialised in metagenomic processing such as \ac{MG-RAST} \cite{Meyer2008} and \ac{CAMERA} \cite{Sun2011} for cellular metagenomes as well as \textsc{Metavir} \cite{Roux2011} and \textsc{VIROME} \cite{Wommack2012} for viral metagenomes. sea star?
These webserver pipelines take on the burden of maintaining the programs, hosting the databases and the computational requirements of the users.
However, these workflows are generally not customisble if the user's needs fall too far outside of that provided. 
For example, requiring different comparative databases, alternative programs or handling sequence data types.
Further, processing massive datasets may still take unfeasibly long time.
cloud computing resources \cite{}
to do this is a realistic time frame can be extremely challenging.
For example, (cite Moore's law).

%Solutions to computational resources are web based servers like MG-RAST, CAMERA, cloud computing.
%Technological bottle necks, shorter reads, so much data that there's not enough compute time, so much data its hard to find a good repository.

% Limitations of bioinformatic tools themselves.Gene prediction needs to get better. de novo assembly, binning.
%De novo assembly need to improve to be able to tackle highly diverse populations.
% eg. Iverson.
%It is heartening that the OLV genome was actually correctly annotated in the minor virion!
% Having high sequencing depth, a simple target population

% Could a metagenome ever be really `closed' metagenome, unless it is the most simple community, it seems difficult to imagine.

%Biological limitations, interpreting of the biology is only as good at the data bases. requires good recording of metadata.
%A lucky tail, discovery of the virophage was only possible with the presence of the Sputnik in 2008. If databases from 2007 were used, it simply would have been an abundant unknown molecule, likely not even recognisably viral.
%Similarly, if dddD and dddL genes were not in the databases at the right time, they would not be seen.
%In some ways metagenomic data still contains an 'unseen majority. Particularly viral metagenomes.
%' the difference is that all this information can be a resource that is reanalysed.
 
%%Metaproteomics
%Highly dependent on having a matching metagenome
%Compared to metagenomics, relatively data poor as a lot of the spectra cannot be matched
%But spectra can be reanalysed

%%Future of -omics studies will have to go beyond just sequencing and sequencing more.
% Need to have studies over long time scales, and finer spatial resolutions.
% Need to marry with emerging technologies.
% Spatial resolution has begun with microfluidics systems (Seymour), 
% digital PCR to associate
% Ultimately, these technologies are tools that we are just learning to use


\subsection{Summary of the outcomes from the work in this thesis}
%This has necessitated the development of new ways to extract biological inferences from complex datasets.
Shotgun metagenomic sequencing is steadily becoming adopted as a standard tool for interrogating microbial communities and, like any tool, has strengths and limitations (cite Eisen 2007, some others).
Development of an epifluorescence microscopy protocol and metaproteomic analysis workflow described in chapter \ref{ch:ace} was instrumental in deriving other measurements of community function alongside metagenomic sequencing.
This work developed on the first integration (read up in that paper of metaproteomic integration) of the two complementary techniques in Antarctic lake communities and laid the groundwork for subsequent work on Organic Lake (chapters \ref{ref:olv} and \ref{ch:org}).
It was a vital part of assessing the functional capacity of entire populations of microbes, almost all of which are still uncultured.
It has generation of hypotheses into the function Ace Lake cumulating in the predictive modelling of the \ac{GSB} community.

The greatest achievement of these projects has been to discover totally new taxa and things about the ecosystem that are testable hypotheses of ecosystem function.
Testing of some of these hypotheses is possible with further molecular methods.
However, a lot of them require adopting new ones like single cell sorting.
Future work that could examine these hypotheses from the discoveries made a detailled below.

%We have used theory to guide our work.

\section{Future work}
\subsection{Dynamics of the over an annual cycle}
It is increasingly clear that metagenomic sampling efforts has often been one hit wonders (cite some paper from Jed Fuhrman).
To truly establish a baseline for the community, what happens over the polar winter needs to be examined.
Using the metagenomic strategies developed in these papers this can happen.
Some of the key question are:
1) how does the community change in winter?
We already see drastic changes from the Palmer ocean samples (cite Tim and Alison's paper). 
Winter samples with no light and how the GSB, synechoccous, algae are affected need to be examined.
We already know ace Lake pyramimonas become bacteriouvorous. 
What strategies are open to the other phototrophs?
Can those changes be detected with functional and metagenomics? yes why not.

What happens to the associated viral populations over the winter?
Do we see strain cycling?
How do the heterotrophs cope? switch to facultative chemoautotrophy?
%Future experiments can look at fluorescence of bchlA

\subsection{Virophages}
Since the discovery of the \ac{OLV} \cite{Yau2011} described in chapter \ref{ch:olv}, two other members of the virophage family have been reported.
The first of these is the Mavirus (for Maverick virus) so named for its evolutionary relationship with the Maverick/Politon class of eucaryotic transposons \cite{Fischer2011}.
Like Sputnik, Mavirus has an absolute requirement for coinfection with a helper virus, in this case \ac{CroV}, in order to replicate.
Also like Sputnik, addition of Mavirus during \ac{CroV} infection leads severe drops in production of \ac{CroV} \cite{Fischer2011}.
The other virophage reported was called Sputnik 2 (its genome is virtually identical to Sputnik) and like Sputnik was also found in association with strain of mimivirus, named lentille virus \cite{Desnues2012}.
Unlike Sputnik, it was found both as a free virion and integrated in the lentille virus genome \cite{Desnues2012}.

The availability of these new virophages in culture affords a new perspective on \ac{OLV} and virophages in general.
In first place, the inability for Mavirus to infect without \ac{CroV} and its detrimental effect on \ac{CroV} just like Sputnik strengthens the assumption that OLV would similarly have a `virophage' phenotype \ac{OLPV}.
This is in part because Mavirus and \ac{CroV} and quite divergent from Sputnik and mimivirus respectively, yet retain the same traits \cite{Fischer2010, Fischer2011}, suggesting the `virophage' phenotype is common to the whole lineage.
Secondly, these cultured virophages offers some mechanistic reasons for why \ac{OLV} would function as the other virophages.
As both mimivirus and \ac{CroV} encode a great deal of their own replication and transcription machinery, including all \textsc{DNA} and \textsc{RNA} polymerase subunits, they do not localise to the nucleus during infection but generate the viral factory in the cytoplasm \cite{LaScola2008, Fischer2011}.
Sputnik and Mavirus replicate exclusively in the giant virus factory and make use of the helper virus replication and transcription systems \cite{LaScola2008, Fischer2011}.
Furthermore, Sputnik and Mavirus share the promoter and other transcriptional control sequences of their helper viruses \cite{Claverie2009, Fischer2011}.
They therefore are primarily parasitising the resources of the helper viruses, which seems a likely cause of reduced helper virus production \cite{Claverie2009, Fischer2011}.
\ac{OLPV}-1 encodes all eight \textsc{RNA} polymerase subunits (see GenBank accession HQ704802) indicating it, like all other members of the mimivirus lineage, would replicate solely in the cytoplasm.
Therefore, it is likely \ac{OLV} would exclusively utilise its helper virus' molecular machinery to similarly detrimental effect.
Ultimately, definitely determining if this is the case for the \ac{OLV} would require isolating it from the environment.

Comparison of three different virophage genomes has raised tantalising questions about virophage physiology, evolution and ecology
All three genomes have share homologues of the Sputnik V20 \ac{MCP}, V3 FstK-HerA DNA packaging ATPase and V9 putative cysteine protease but otherwise are quite divergent.
%Sputnik and Mavirus additionally share a the V13 primase/superfamily 3 helicase and the V14 Zn-ribbon domain containing protein.
%While Sputnik and \ac{OLV} share the V18/19 minor virion protein, V1 primase-polymerase containing protein and V21 hypothetical protein.
Mavirus, in particular, is in many ways more similar to the Maverick/Politon transposable elements in the possession of a retroviral-family integrase that is absent in Sputnik and \ac{OLV} \cite{Fischer2011}.
This was theorised to have been separately acquired as a way to stabilise the relationship between the ancestral Mavirus and its cellular host as integration of a virophage could perhaps serve as a defence against infection of the giant helper virus \cite{Fischer2011}.
In support of this, Mavirus can be independently phagocytosed by \emph{Cafeteria roenbergensis} in the absence of \ac{CroV} -- potentially to reduce the severity of a subsequent \ac{CroV} infection \cite{Fischer2011}.
As yet Mavirus has not as yet been reported to be integrated in the host \emph{Cafeteria roenbergensis}.
In contrast, Sputnik seems to associate directly with the fibrils that coat the mimivirus virion, not the host \emph{Acanthamoeba} cell \cite{Boyer2011}.
These two modes of infection, that is a virophage--host interaction followed by helper \emph{vs}. helper--virophage co-infection of host cell, would lead to distinct effects on the population dynamics in the environment.
Detection of \ac{OLV} genome as a separate circular molecular in the 0.8-- 0.1 \textmu{}m size fraction (Chapter\ref{ch:olv}) indicates it is somehow in association with larger particles.
Determination of the mode of infection for \ac{OLV} would improve predictive modelling of \ac{OLV} driven impacts on algal blooms.
This could be achieved by observation of \ac{OLV}-\ac{OLPV} infections in culture or flow cytometric sorting of sample populations of potential hosts and \ac{OLPV} from the environment and screening of the presence of \ac{OLV}.
The latter experiment would also establish fundmental properties of \ac{OLV} such as the identity of hosts and helper viruses, proportions of infected cells and proportions of \ac{OLPV} infections that include an \ac{OLV}.
 
The discovery that Sputnik 2 can integrate into its helper lentille virus suggests a similar interaction could occur between \ac{OLV} and \ac{OLPV}.
This is because integration of Sputnik 2 is localised to a 352 bp region shared by Sputnik 2 (corresponding to the Sputnik V6 gene), lentille virus and mamavirus  \cite{Desnues2012} that encodes a collagen-like repeat-containing protein.
\ac{OLV} and \ac{OLPV} shared a homologous collagen-like repeat-containing protein (Chapter\ref{ch:olv}) that may similarly function as a site of integration.
As yet, the conditions and mechanism by which Sputnik 2 integrates is unknown.
From the metagenomic sequence, \ac{OLV} assembled as a separate molecule (Chapter\ref{ch:olv}).
Screening of metagenomic assemblies from other sampling dates, such as Ace lake and the 2008 Organic Lake samples, for \ac{PV} genomes with integrated virophages can be performed to see if this occurs in the environment at different times.
Again, obtaining isolates or assembling more \ac{PV} genomes can determine if this is a common trait in other virophage systems.
If integration occurs between \ac{OLV} and \ac{OLPV} it would have interesting implications for their natural dynamics.
One interesting possibility is that integration of virophages may function analogously to lysogeny in bacteriophages if integration lead to a `dormant' state for the virophage.
In this scenario, virophages would then integrate into their helper virus when helper virus densities are low to avoid driving their helpers to extinction.
On the other hand, if integration does not entail dormancy of the virophage, it could function as a means to ensure transmission.

%%%OLV distribution and evolution questions
%virophages seem to attack only mimivirus relatives.
%How common are they in the environment?
%They seem like an ancient lineage, how widespread are they? Do they infect other NCLDV


\subsection{Functional analyses in the Organic lake community}
%Need to show that AAnP genes are expressed with RT-PCR, proteomics

%Cloning of the dddD, dddL genes into appropriate host to show activity.
%Look for inputs into the lake.
%Is allochtonous inputs feeding the lake?
%Need to show the dynamics of the lake
%Need to look at the rest of viral community 

\section{Assessment of the value of molecular antarctic research}
%Molecular data can be continued to be mined! 
%Serves as a record for prosterity
%Metagenomics is not enough! There needs to be functional data, single cell data, biochemical data.

