\chapter{Virophage Control of Antarctic Algal Host--Virus Dynamics}
\label{ch:olv}

%-----------------------------------------------------------------------------------------------------
\section*{Co-authorship Statement}
\addcontentsline{toc}{section}{Co-authorship Statement}

A version of this chapter has been published as:\\

\textbf{Sheree Yau}, Federico M. Lauro, Matthew Z. DeMaere, Mark V. Brown, Torsten Thomas,
Mark J. Raftery, Cynthia Andrews-Pfannkoch, Matthew Lewis, Jeffery M. Hoffman, John A. Gibson, and
Ricardo Cavicchioli (2011)
Virophage control of antarctic algal host--virus dynamics.\\
\textit{\underline{Proceedings of the National Academy of Sciences USA}}
\textbf{108}: 6163--6168.\\

Contributions to this publication by other researchers is as follows.
Research was designed by Federico Lauro, Mark Brown, Torsten Thomas, John Gibson and Ricardo Cavicchioli.
Sample collection was performed by Federico Lauro, Mark Brown, Torsten Thomas, Jeffery Hoffman and Ricardo Cavicchioli.
DNA extraction and clone library preparation of 2006 samples was performed by Cynthia Andrews-Pfannkoch and Jeffery Hoffman of the J. Craig Venter Institute.
DNA sequencing quality control was performed by Matthew Lewis of the J. Craig Venter Institute.
Metagenomic sequence processing, global assembly and annotation was performed by Matthew DeMaere.
Assistance in mass spectronomy and mass spectra analysis was provided by Mark Raftery.
Assistance in analysis of Eucarya taxonomy was provided by Mark Brown.
Analysis of virophage abundance over time was performed by Federico Lauro.

Apart from these contributions, I performed all other data analyses and interpretations.
\newpage

%----------------------------------------------------------------------------------------------

\section{Abstract}

Viruses are abundant ubiquitous members of microbial communities, and in the marine environment affect population structure and nutrient cycling by infecting and lysing primary producers. 
Antarctic lakes are microbially dominated ecosystems supporting truncated food webs where viruses exert a major influence on the microbial loop. 
Here we report the discovery of a new virophage (relative of the recently described Sputnik virophage) that preys on phycodnaviruses that infect prasinophytes (phototrophic algae). 
By performing metaproteogenomic analysis on samples from Organic Lake, a hypersaline meromictic lake in Antarctica, complete virophage and near-complete phycodnavirus genomes were obtained. 
By introducing the virophage as an additional predator of a predator-prey dynamic model we determine that the virophage stimulates secondary production through the microbial loop by reducing overall mortality of the host and increasing the frequency of blooms during polar summer light periods. 
Virophages remained abundant in the lake two years later, and were represented by populations with a high level of major capsid protein sequence variation (25--100\% identity). 
Virophage signatures were also found in neighbouring Ace Lake (in abundance), and in two tropical lakes (hypersaline and fresh), an estuary, and an ocean upwelling site. 
These findings indicate that virophages regulate host-virus interactions and influence overall carbon flux in Organic Lake, and play previously unrecognised roles in diverse aquatic ecosystems.
\newpage

%---------------------------------------------------------------------------------------------

\section{Introduction}
It has been known for at least 20 years that viruses frequently infect and lyse marine primary producers causing up to 70\% of cyanobacterial mortality \cite{Proctor1990,Suttle1990}.
Eucaryotic phytoplankton are preyed upon by large dsDNA phycodnaviruses (PVs) causing bloom termination in globally distributed species (3,6).
Elevated levels of dissolved organic carbon (DOC) (7) and numbers of heterotrophic bacteria (8-10) occur during algal blooms indicating that viral lysis of eucaryotic algae stimulates secondary production. 
Viruses also suppress host populations at concentrations below bloom-forming levels, with abundance being controlled by the efficiency and production rates of the infecting viruses (11, 12). 

Antarctic lakes are microbially dominated ecosystems supporting few, if any metazoans in the water column (13, 14). 
In these truncated food webs, viruses are expected to play an increased role in the microbial loop (15). 
Low complexity Antarctic lake systems are amenable to whole community based molecular analyses where the role that viruses play in microbial dynamics can be unravelled (14). 
Attesting to this, a metagenomic study of Lake Limnopolar, West Antarctica uncovered a dominance of eucaryotic viruses and ssDNA viruses previously unknown in aquatic systems (16). 

We established a metaproteogenomic program for Organic Lake (68$^{\circ}$27$'$23.4$''$S, 78$^{\circ}$11$'$22.6$''$E), which is located in the Vestfold Hills, East Antarctica, in order to functionally characterize its microbial community. 
Organic Lake is a shallow (7 m) hypersaline ($\approx$230 g L$^{-1}$ maximum salinity) meromictic lake with a high concentration of dimethylsulphide ($\approx$120 $\mu$g $^{-1}$) in its anoxic monimolimnion (17, 18). 
Water temperature at the surface of the lake can vary from $-$14 to $+$15$^{\circ}$C while remaining sub-zero at depth (19, 20). 
The lake is eutrophic, with organic material sourced both from autochthonous production and input from penguins and terrestrial algae. 
The high concentrations of organic material reflect slow breakdown in the highly saline lake water. 
The salt in the lake was trapped along with the marine biota when the lake was formed due to falling sea level c. 3,000 y BP (21, 22). The lake sediment has both low species diversity (Shannon-Weaver diversity: 1.01) and richness (Chao non-parametric index: 32 $\pm$ 12) (23). 
Unlike high latitude lakes, viral abundance has been reported to increase with trophic status (15) and with salinity in Antarctic lakes (24). 

Here we report the analysis of the surface water of Organic Lake, highlighting the presence of a relative of the recently described Sputnik virophage, a small eucaryotic virus that requires a helper \textit{Acanthamoeba polyphaga} mimivirus (APMV) to replicate (25). 
From metagenomic DNA, a complete Organic Lake virophage (OLV) genome was constructed (the second virophage genome to be described), and near-complete genomes of its probable helper Organic Lake phycodnaviruses (OLPVs).

%------------------------------------------------------------------------------------------

\section{Materials and Methods}

\subsection{Samples and DNA Sequencing}
Water samples collected from Organic Lake were: 

\begin{enumerate}
\item Surface water from the eastern side of the ice-free lake (68$^{\circ}$27$'$25.48$''$S, 78$^{\circ}$11$'$28.06$''$E) December 24, 2006.
\item A depth profile collected through a 30 cm hold drilled through the surface ice above the deepest point in the lake (68$^{\circ}$27$'$22.15$''$S, 78$^{\circ}$11$'$23.95$''$E), November 10, 2008. 
\item surface water from the north-east side of the partially ice-covered lake (68$^{\circ}$27$'$21.02$''$S 78$^{\circ}$11$'$42.42$''$E), December 12, 2008. 
\end{enumerate}

Samples were sequentially filtered through a 20 $\mu$m pre-filter and biomass captured onto 3.0, 0.8 and 0.1 $\mu$m membrane filters as described previously (1, 2). 
The samples from 2008 also included 50\% (v/v) RNAlater. 
DNA extraction, sequencing and quality validation was performed as previously described (1, 2). 
DNA sequencing was performed at the J. Craig Venter Institute in Rockville, MD, USA.  

\subsection{Transmission Electron Microscopy}
Unfiltered Organic Lake surface water from December 24, 2006 (fixed on-site in 1\% (v/v) formalin) was concentrated and a solvent exchange performed with sterile filtered ammonium acetate solution 1\% (w/v) using a 50 kDa cut-off Microcon centrifugal filter device (Millipore) according to the manufacturerâ€™s instructions. 
Formvar coated 200 mesh copper grids were floated on a droplet of sample for 30 min, excess liquid wicked off and the grid negatively stained for 30 s with uranyl acetate 2\% (w/v). 
The sample was visualised using a JEOL1400 transmission electron microscope at 100 kV at 150,000 to 250,000 $\times$ magnification.

\subsection{Metagenomic Assembly and Annotation}
Mosaic metagenomic assemblies were generated as previously described (1, 2). 
For the 0.1 $\mu$m Organic Lake 2006 sample, assembly was a hybrid of Sanger and 454 read data (Table S1). 
For all other sample size fractions, runtime parameters used were standard for 454 sequencing data. 
Low GC ($\ge$ 51\%) scaffolds $>$ 10 kb from the 0.1 $\mu$m 2006 assembly had high coverage ($>$ 45 $\times$) indicating these were from the dominant taxa. 
One of these scaffolds was binned as virophage and the rest as PV. 

To further separate the OLPV types and assess the completeness of their genomic content, highly conserved single copy PV orthologues were identified as follows. 
An all against all BLASTp search was conducted with protein sequences from the ten available PV genomes 
(\textit{Acanthocystis turfacea Chlorella} virus 1, PbCV-1, PbCV AR158, PbCV FR483, PbCV NY2A, \textit{Emiliania huxleyi} virus 86, \textit{Ectocarpus siliculosus} virus 1, \textit{Feldmannia} sp. virus, \textit{Ostreococcus} virus 5, \textit{Ostreococcus tauri} virus 1) and APMV (which was included as a close PV relative). 
BLASTp results were parsed and clustered using orthoMCL V1.4 (3, 4). 

Pairs of each orthologue were located on eight of the PV scaffolds. 
The location of each orthologue pair had a complementary distribution so the eight scaffolds were able to be sorted unambiguously into two strains (OLPV-1 and OLPV-2). 
OLPV-1 ribonucleotide reductase $\alpha$-subunit appeared as duplicated on different scaffold ends, likely as an artefact of its proximity to an assembly break point. 
The remaining high coverage scaffolds were searched for predicted proteins present in one OLPV strain but not in the other and assigned to the strain in which it was absent. 
Comparison of OLPV-1 and OLPV-2 scaffolds was performed using tBLASTn of concatenated scaffolds from each strain and visualised using the Artemis Comparison Tool (ACT) (5). 
DNA sequence data is available in Genbank and CAMERA (http://web.camera.calit2.net).

\subsection[Genome Completion and Annotation]{Organic Lake Virophage Genome Completion and Annotation}
The high coverage (77 $\times$), large number of Sputnik homologues that encode essential functions and length of the putative OLV scaffold from the 0.1 $\mu$m 2006 hybrid assembly indicated it was a near-complete genome. 
Reads from this scaffold were reassembled at high stringency and visualised using Phred/Phrap/Consed (6) to complete the sequence. 
Mate-pair data indicated a circular molecule and primers were designed to span the ends of the scaffold and sequence across the gap (Table S5). 
Touch-down PCR was performed with eDNA from 0.1 $\mu$m 2006 sample, the product used for nested PCR and the final product was cloned and sequenced. 
The complete genome was manually annotated and visualised using Artemis (7). 
Translated ORFs (minimal size 120 amino acids) were compared (BLASTp) to GenBank, to the all metagenomic ORF peptide database on CAMERA (http://web.camera.calit2.net) and to predicted proteins from OLPV-1 and OLPV-2 scaffolds. 
Comparisons between the OLV genome and OLPV-1 /OLPV-2 were performed with tBLASTn and visualised using ACT (5). 

\subsection{Phylogenetic Analysis}
Translated amino acid sequences from viral marker genes of interest were retrieved from the 0.1 $\mu$m 2006 metagenomic assemblies from this study, GenBank and CAMERA all metagenomic reads ORF peptide database. Homologous sequences were aligned using MUSCLE v3.6 (8). 
Neighbour-joining analysis, test for clade support (bootstrap analysis 2000 replicates) and tree drawing was performed with Molecular Evolutionary Genetics Analysis (MEGA) software v4 (9). 
Maximum likelihood analysis (JTT substitution model) and test for clade support (aLRT analysis) was performed with PhyML (10) and the tree visualised using MEGA. 
18S rRNA gene sequences were retrieved from reads of all filter sizes, compared (BLASTn, e-value $<$ 1.0e$-$5) to the SILVA100 SSURef database, aligned and phylogeny performed using ARB as previously described (1, 2). 
The abundance and similarity of virophages in all lake samples and filter sizes was estimated using BLASTp (evalue $<$ 1.0e$-$5) to search using the OLV MCP sequence against a database of proteins predicted from sequencing reads. 
The database was generated as previously described (1) and the percent identity of the BLAST hit was used as a proxy for species similarity. 

\subsection{Metaproteomic Analysis}
Metaproteomics of proteins from the 0.1 $\mu$m filter from 2006 was performed as previously described (1, 2), with minor modifications. 
The protein sequence database was generated by combining ORFs from the 3.0, 0.8 and 0.1 $\mu$m mosaic assemblies with 130,581 sequences in the database. 
Scaffold 3.0 (Proteome Software Inc.) was used to validate MS/MS based peptide and protein identifications. 
Protein identification data is available in Table S2. 

\subsection[Algal Host--Virus and Virophage Dynamics]{Model of Algal Host--Virus and Virophage Dynamics}
To model the effect a virophage would have on algal Pyramimonas algal host populations in Organic Lake, modified Lotka-Volterra equations were used describing the OLV as a predator of predator OLPV. 
The original equations are given by:

\begin{equation}
\frac{\mathrm{d}A}{\mathrm{d}t}=\alpha A - \varepsilon PA
\label{eqn:lokprey}
\end{equation}

\begin{equation}
\frac{\mathrm{d}P}{\mathrm{d}t}= \theta PA - \mu P
\label{eqn:lokpred}
\end{equation}

Where:
\begin{description}
\item[$A$] is the number of \textit{Pyramimonas} (prey).
\item[$P$] is the number of OLPV (predator).
\item[$\alpha$] is the specific growth rate of the prey.
\item[$\theta$] is the specific production rate of the predator.
\item[$\varepsilon$] is the rate of predator mediated death of prey.
\item[$\mu$] is the specific decay rate of the predator.
\end{description}

Equation \ref{eqn:lokprey} describes the change in \textit{Pyramimonas} abundance and equation \ref{eqn:lokpred} the change in OLPV abundance in the absence of OLV.
In the presence of OLV, \textit{Pyramimonas}, OLPV and OLV dynamics are described by the following equations:

\begin{equation}
\frac{\mathrm{d}P}{\mathrm{d}t}= \theta PA - \mu P - \omega PV
\label{eqn:pred}
\end{equation}
\begin{equation}
\frac{\mathrm{d}V}{\mathrm{d}t}=\beta PV - \gamma V
\label{eqn:viro}
\end{equation}

Where:
\begin{description}
\item[$V$] is the number of teh OLV (predator of predator).
\item[$\omega$] is the rate of OLV mediated reduction in OLPV infective particles.
\item[$\beta$] is the production rate of OLV.
\item[$\gamma$] is the decay rate of OLV.
\end{description}

Equation \ref{eqn:pred} is a modified version of equation \ref{eqn:lokpred} which includes the effect of OLV on the change in abundance of OLPV.
Equation \ref{eqn:viro} describes the growth properties of OLV as a predator of OLPV.
Values for the variables for the solution shown (Fig. 4) were as follows: initial prey (10), predator (1) and predator of predator (10) numbers, ~$\alpha=$ 0.1, ~$\theta=$ 0.0015, ~$\varepsilon=$ 0.01, ~$\mu=$ 0.01, ~$\omega=$ 0.01, ~$\beta=$ 0.015 and ~$\gamma=$ 0.15. 
COmplex PAthway Simulator (COPASI) software (11) was used to simulate prey, predator and predator of predator dynamics using the deterministic (LSODA) method.
%-----------------------------------------------------------------------------------------


\section{Results and Discussion}
\subsection{Dominance of Phycodnaviruses in Organic Lake}
Water samples from Organic Lake were collected December 2006 and November and December 2008, and microbial biomass collected onto 3.0, 0.8 and 0.1 $\mu$m membrane filters as described previously (14). 
A large proportion of shotgun sequencing reads (96.2\%) from the 0.1 $\mu$m size fraction of the 2006 Organic Lake metagenome (Table S1) had no significant hits to sequences in the RefSeq database 
(tBLASTx with e-value $<$ 1.0e$-$3, minimum alignment length: 60 bp, minimum identity: 60\%). 
The degree of assembly was high, with 77\% of reads forming part of a scaffold, indicating the sample contained a few abundant taxa of minimal diversity. 
Forty-five scaffolds were longer than 10 kb; the five longest ranged from 70 to 171 kb. 
GC content and coverage were used to separate scaffolds into taxonomic groups (Fig. S1). 
A broad division was evident between low ($\le$ 41\%) and high ($\ge$ 51\%) GC scaffolds suggesting they constituted two taxonomic groups. 
All scaffolds in the high GC group that could be assigned contained phage homologues, as did the one exceptional low GC scaffold. 
The low coverage in the high GC group showed bacteriophages were not abundant in the 0.1 $\mu$m fraction. 
These scaffolds were not analyzed further. 
The low GC scaffolds with confident assignments contained sequences matching conserved PV or APMV proteins. 
These PV-related scaffolds comprised 60\% of assembled reads demonstrating that OLPVs were numerically dominant in the 0.1 $\mu$m fraction. 
Transmission electron microscopy (TEM) revealed the presence of virus-like particles with the dimensions and structure typical of PVs (Fig. 1A).

Within the low GC group, scaffolds separated into a high coverage ($>$ 45$\times$) group, including the five longest scaffolds, and a low coverage ($<$ 22$\times$) group. 
Two of the scaffolds in the high coverage group and one in the low coverage group contained the PV marker DNA polymerase B (DPOB). 
The two high coverage DPOB share 76\% amino acid identity and both share ~$\approx$57\% identity to the low coverage DPOB. 
DPOB is single-copy throughout the nucleo-cytoplasmic large DNA virus (NCLDV) family to which PVs belong (26,27), 
demonstrating that the Organic Lake surface waters contained two closely related abundant PV types (DPOB1) and (DPOB2), and a more distantly related lower abundance type (DPOB3). 

Phylogenetic analysis clustered Organic Lake DPOB with unclassified lytic marine PV isolates that infect the prymnesiophytes \textit{Chrysochromulina ericina} (CeV1) and \textit{Phaeocystis pouchetii} (PpV),
 the prasinophyte \textit{Pyramimonas orientalis} (PoV) (4,28), and uncultured marine PVs related to APMV (29, 30) (Fig. 2A and Fig. S2). 
As the host range of PVs broadly correlates with DPOB phylogeny (31, 32), OLPV would infect prasinophytes or prymnesiophytes. 
The most probable host is the prasinophyte, \textit{Pyramimonas} (no prymnesiophyte 18S rRNA gene sequences were present in any size fraction of the Organic Lake metagenome) (Fig. S3).

Supporting the presence of more than one PV, pairs of single-copy PV orthologues 
(ribonucleotide reductase alpha and beta subunits, VV A32R virion packaging helicase, PBCV1 A482R-like putative transcription factor, VV D5 ATPase and VLTF2 family transcription factor) 
were identified in the high coverage scaffolds that shared an average of 81\% percent amino acid identity.
 Based on the positions of single copy genes on the scaffolds and the percent identity between them, the high coverage scaffolds were grouped into two strains designated OLPV-1 and OLPV-2 according to their DPOB phylogeny (Fig. 2A and Fig. S2).
 The remaining high coverage scaffolds were assigned to either strain, resulting in two near-complete genomes of $\approx$300 kb each (Fig. 2C), 
that are within the range of other sequenced PV genomes (155-407 kb). 
In addition, several OLPV genomic fragments contained PV homologues in high coverage scaffolds that could not be confidently assigned to either strain. 

Both OLPV strains contain a PpV-like major capsid protein (MCP) designated MCP1 and another unique MCP designated MCP2 (Fig. 2B and Fig. S4). 
Both OLPV MCP1s were identified in the metaproteome (Fig. 2C and Table S2) but MCP2 was not. 
In addition to MCPs, the metaproteome contained a range of abundant structural proteins and others more likely to be packaged in the virion (e.g. chaperone), that were expressed by OLPV-1, OLPV-2 and/or an OLPV genomic fragment (Fig. 2C and Table S2). 
These data suggest that MCP1 is the major structural protein, and that both OLPV-1 and OLPV-2 were in a productive cycle in the lake at the time of sampling. 

\subsection{Complete Genome of an Organic Lake Virophage}
Sputnik is a small (50 nm) icosahedral satellite virus of mamavirus (a new strain of APMV). 
It was termed a ``virophage'' because co-infection with Sputnik is deleterious to the mamavirus, resulting in abnormal virions and a decrease in mamavirus infectivity (25). 
One 28 kb scaffold in the low GC high coverage group had six out of 38 predicted proteins homologous to Sputnik virophage proteins (Fig. 3 and Table S3), and one PV homologue. 
The scaffold had a low GC content (~$\approx$30\%), similar to the Sputnik genome, and was larger in size (28 kb vs 18 kb for Sputnik). 
Using PCR and sequencing, the scaffold was found to represent a complete circular virophage genome (the Sputnik genome is also circular). Virus-like particles resembling Sputnik in size and morphology were identified by TEM (Fig. 1B).

Sputnik homologues present in the Organic Lake scaffold included the V20 MCP, V3 DNA packaging ATPase, V13 putative DNA polymerase/primase and others of unknown function (V9, V18, V21 and V32) (Fig. 3 and Table S3). 
The Organic Lake virophage (OLV) is distinct to Sputnik as proteins share 27-42\% amino acid identity (28\% MCP identity). 
OLV proteins include OLV9, the homologue of Sputnik V20 MCP, and OLV8, a fusion of the uncharacterised V18 and minor virion protein V19 from Sputnik (Fig. 3 and Table S3). 
The large number of homologues, including genes that fulfill essential functions in Sputnik (V20, V3 and V13), indicate that OLV and Sputnik have physiological similarities. 

\subsection{Gene Exchange Between Virophage and Phycodnaviruses}

As PVs are related to APMV (27) and are abundant in Organic Lake, it stands to reason that OLPV is the helper of OLV. 
In the OLV genome, OLV12 is a \textit{Chlorella} virus-derived gene, indicating that gene exchange has occurred between OLV and PVs (the function of OLV12 is discussed below). 
Similar observations were made for Sputnik, which carries four genes (V6, V7, V12 and V13) in common with the mamavirus, indicative of gene exchange between the viruses and possible co-evolution (25). 
As the V6, V7, V12 and V13 proteins have been associated with virophage-helper specificity, we reasoned that the functional analogues in OLV would have highest identity to proteins from its helper virus, rather than Sputnik. 

By comparing OLV and OLPV, a 7,408 bp region was identified in OLV encoding five proteins (OLV17-22) with identity (32--65\%) to sequences in both OLPV-1 and OLPV-2 (Fig. 2C, Fig. S5 and Table S3). 
OLV20 and OLV13 are collagen triple-helix-repeat-containing proteins, analogous to Sputnik collagen-like proteins (V6 and V7) involved in protein-protein interactions in the APMV virus factory. 
Sputnik can replicate with either mamavirus or APMV as a helper, although coinfection rates are higher with the mamavirus. 
V6 is the only protein with higher identity (69\%) to mamavirus than APMV (42\%) (25). 
Since OLV20 has equivalent identity (63\%) with OLPV-1 and OLPV-2, it appears that OLV may be capable of interacting with both OLPV strains. 
Also within the conserved region, OLV22, is a 141 aa protein of unknown function that only matches sequences from OLPV and the Global Ocean Sampling (GOS) expedition (Table S3). 
Similar to OLV22, Sputnik V12 is a small protein (152 aa) of unknown function with high identity to APMV, and both may mediate a specific helper-virophage interaction. 
Other genes in this region of OLV can be mapped to OLVP, including a putative transmembrane protein (OLV17) and paralogous phage tail fibre repeat containing proteins, OLV18 and OLV19. 
Analogous to the collagen-like proteins, OLV19 and OLV20 probably facilitate interactions between helper and virophage. 

OLV12 (which is unique to OLV) consists of a C-terminal domain present in conserved hypothetical \textit{Chlorella} virus proteins and an N-terminal domain most closely related to class 3 lipases that may confer OLV selectivity to a PV. 
OLV12 may function similarly to the Sputnik V15 membrane protein in modifying the APMV membrane (25). 
The Sputnik V13 consists of a primase domain and SF3 helicase domain related to NCLDV homologues, involved in DNA replication. 
The helicase domain of OLV25 and V13 are similar, although the primase domain is more similar to a protein from \textit{Ostreococcus lucimarinus}, implying a past association of OLV with a prasinophyte alga host. 

Genes unique to OLV point to adaptations specific to its helper-host system. 
Most notably, OLV possesses a N6 adenine-specific DNA methyltransferase, as does OLPV. 
In OLPV-1, genes for a bacterial type I restriction modification (RM) system are adjacent to a gene encoding a type I methylase-S target recognition domain protein, and upstream of a DNA helicase distantly related to type III restriction endonuclease (RE) subunits. 
A large number of \textit{Chlorella} virus genomes have both 5mC and 6mA methylation (33), and several contain functional RM systems (34). 
The prototype \textit{Chlorella} virus PbCV-1 possesses REs packaged in the virion for degrading host DNA soon after infection (35). 
In contrast to OLV/OLPV, DNA methyltransferases are absent in both Sputnik and APMV, indicating that the N6 adenine-specific DNA methyltransferase has been selected in OLV to reduce endonucleolytic attack mediated by OLPV. 

\subsection[Virophage in Algal Host--Phycodnavirus Dynamics]{Role of Virophage in Algal Host--Phycodnavirus Dynamics}
The presence of the virophage adds an additional consideration to the microbial loop dynamics. 
In batch amoeba cultures, co-infection of amoeba with APMV and Sputnik causes a 70\% decrease in infective AMPV particles and a 3-fold decrease in lysis (25). 
To test how OLV affects OLPV and host population dynamics, we modelled the OLV as an additional predator of a predator in a Lotka-Volterra simulation (Fig. 4). 
In the model, the effect of virophage is robust, with equilibrium solutions across a wide range of parameter values (Fig. 4 shows one equilibrium solution). 
By decreasing the number of infective OLPVs, the presence of OLV shortens the recovery time of the host population (Fig. 4C) and shifts the orbit away from the axis (Fig. 4D). 
The model reveals that the virophage stimulates the flux of secondary production through the microbial loop by reducing overall mortality of the host algal cell following a bloom, and by increasing the frequency of blooms during the summer light periods. 
Antarctic lake systems have evolved mechanisms to cope with long light-dark cycles (14) and shortened trophic chains. 
In Organic Lake and similar systems, a decrease in PV virulence may be instrumental in maintaining stability of the microbial food web. 

\subsection{Ecological Relevance of Virophages in Aquatic Systems}
Metagenomic analysis of Organic Lake samples taken two years later in November (when the lake was ice covered) and December 2008 (partially ice-free) revealed sequences with 99\% amino acid identity to OLV MCP indicating persistence of OLV in the ecosystem (Fig. 5 and Table S4). 
In addition, sequences with lower identity (25--90\%) were detected, particularly in December, demonstrating Organic Lake virophages are highly diverse but OLV remained the dominant type. 

From surface water samples of nearby Ace Lake (meromictic, surface 2\% salinity), a large number of sequences were obtained that matched both the OLV MCP (Fig. 5 and Table S4) and PVs (14). 
All Ace Lake size fractions contained matches to OLV MCP, some with high identity (80--100\%) and the majority with greater variation (25--80\% identity) (Fig. 5 and Table S4). 
In contrast to Organic Lake where the largest number of matches was to the 0.1 $\mu$m size fraction, the majority of Ace Lake sequences were from the larger fractions (Fig. 5 and Table S4). 
This indicates the Ace Lake virophages were associated with host cells during sampling, or possibly with helper viruses that are larger than the OLPVs. 

Extending the OLV MCP search to the GOS data revealed matches (25-28\% identity) to sequences from the hypersaline Punta Cormorant Lagoon (Floreana Island, Galapagos), an oceanic upwelling near Fernandina Island (Galapagos), Delaware Bay estuary (NJ, USA), and freshwater Lake Gatun (Panama) (Table S4). The phylogenetic analysis of a conserved 103 amino acid region of the MCPs revealed a number of clusters, with Sputnik clustering with virophage sequences from Ace Lake that had low identity (22\%) to OLV MCP (Fig. 5 and Fig. S4). 
To improve searches for virophages and better understand their physiology and evolution, it will be valuable to target more genomes (e.g. the Ace Lake 167858124 relative with 40\% MCP identity to Sputnik) and determine which genes are core to virophages and what relationship exists between genome complement and MCP identity.
 
In view of the implications of the virophage modelling (Fig. 4), the abundance and persistence of OLV in Organic Lake (Fig. 5, Table S4), and the presence of diverse virophage signatures in a variety of lake systems (fresh to hypersaline), an estuary, an ocean upwelling site and a water cooling tower (Sputnik), our study indicates that numerous types of virophages exist and play a previously unrecognised role in regulating host--virus interactions and influencing ecosystem function in aquatic environments. 

%------------------------------------------------------------------------------------------------------

\section{Acknowledgements}
We thank Craig Venter, John Bowman, Louise (Cromer) Newman, Anthony Hull, John Rich and Martin Riddle for providing helpful discussion and logistical support associated with the Antarctic expedition, and Lisa Ziegler for discussion about marine viruses. 
We acknowledge technical support for computing infrastructure and software development from Intersect, and in particular assistance from Joachim Mai. 
This work was supported by the Australian Research Council and the Australian Antarctic Division. 
Funding for sequencing was provided by the Gordon and Betty Moore Foundation to the J. Craig Venter Institute. 
Mass spectrometric results were obtained at the Bioanalytical Mass Spectrometry Facility within the Analytical Centre of the University of New South Wales. 
This work was undertaken using infrastructure provided by NSW Government co-investment in the National Collaborative Research Infrastructure Scheme. 
Subsidized access to this facility is gratefully acknowledged. 
We thank Jenny Norman from the UNSW Electron Microscopy Unit her assistance in generating images. 
